src/xrGame/DiscordRPC.cpp

```diff
@@ -0,0 +1,35 @@
+#include "stdafx.h"
+#include "DiscordRPC.h"
+#include <ctime>
+
+DiscordRPC* g_DiscordRPC;
+
+void DiscordRPC::Init()
+{
+    DiscordEventHandlers Handle;
+    memset(&Handle, 0, sizeof(Handle));
+    Discord_Initialize("869824972464488510", &Handle, 1, NULL);
+}
+
+DiscordRPC::~DiscordRPC()
+{
+    Discord_Shutdown();
+    Discord_ClearPresence();
```

Comment:
Please, make the code symmetric: move deinitialization code to `Deinit` function.